fiwareskuld/phase/phase0_generate_community_userlist_resources.py

```diff
@@ -0,0 +1,139 @@
+from fiwareskuld.utils import log
+from fiwareskuld.utils import rotated_files
+from fiwareskuld.users_management import UserManager
+from fiwareskuld.conf import settings
+
+logger = log.init_logs('phase0')
+
+
+class CommunityUsers:
+    def __init__(self):
+        """Constructor. Create a keystone client"""
+        clients = osclients.OpenStackClients()
+        clients.override_endpoint(
+            'identity', clients.region, 'admin', settings.KEYSTONE_ENDPOINT)
+        self.keystoneclient = clients.get_keystoneclientv3()
+        self.protected = set()
+
+    def generate_community_users_regions(self):
+        """
+        It generates a file with the community users and the regions
+        where they have access.
+        :return: nothing
+        """
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_community_users()
+
+        self._get_regions_users(community_users, "community_users_regions.txt")
+
+    def generate_expired_community_users_regions(self):
+        """
+        It generates a file with the expired community users and the regions
+        where they have access.
+        :return: nothing
+        """
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_list_expired_community_users()
+
+        self._get_regions_users(community_users, "expired_community_users_regions.txt")
+
+    def generate_community_users_resources(self):
+        """
+        It generate a file with the community expired users.
+        :return: nothing
+        """
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_community_users()
+
+        self._get_community_resources(community_users, 'community_users_regions_resources.txt')
+
+    def generate_expired_community_users_resources(self):
+        """
+        It generate a file with the community expired users resources.
+        :return: nothing
+        """
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_list_expired_community_users()
+
+        self._get_community_resources(community_users, 'expired_community_users_regions_resources.txt')
+
+    def _get_community_resources(self, users, file):
+        user_manager = UserManager()
+
+        with open(file, 'w') as users_to_delete:
+            for user in users:
+                users_to_delete.write(user.id + " " + user.name + " " + user.community_started_at + " ")
+                regions = user_manager.get_regions(user)
+                if regions and "PiraeusU,Hannover,Spain2,Karlsk" in regions:
+                    users_to_delete.write("Projects is not enabled\n")
+                    continue
+                resources = user_manager.get_user_resources_regions(user)
+
+                for resource in resources:
+                    if type(resources) is dict:
+                        if "SophiaAntipolis2" in resource:
+                            users_to_delete.write("Region SophiaAntipolis2")
```

Comment:
So, you mean an exception in the management of service endpoints? In that case should not be better to manage an exception and not introduce an specific case for one region that could be reproduced in future in other regions?